core/src/main/java/nl/nn/adapterframework/configuration/ConfigurationDigester.java

```diff
@@ -234,6 +234,11 @@ public void parseAndResolveEntitiesAndProperties(ContentHandler digester, Config
 		handler = new OnlyActiveFilter(handler, appConstants);
 		handler = new ElementPropertyResolver(handler, appConstants);
 
+		String rewriteLegacyClassNames = appConstants.getProperty("migration.rewrite.legacyClassNames", "false");
+		if (rewriteLegacyClassNames.equalsIgnoreCase("true") || rewriteLegacyClassNames.equalsIgnoreCase("!false")) {
```

Comment:
Eigenlijk moeten we dat in de PropertyLoader stoppen, is ook een issue van #3793.

Voor hier iig `PropertyLoader.getBoolean` dan gebruiken we ten minste overal de juiste methodes. Kunnen we ze in een later PR hernoemen.

Reply:
NB: Deze parameter genaamd `appConstants` heeft eigenlijk als type `Properties`, niet `AppConstants`. Daarom dat ik geen method `getBoolean()` kon vinden...

`AppConstants` kan niet worden gebruikt in de tests voor de `ConfigurationDigester`. Misschien kan `PropertyLoader` wel.

Reply:
Aha, ik snap hem. Volgens mij is dat vooral voor tests zo. Misschien wel dan de variable hernoemen naar properties, of er een propertyloader van maken 😮

Reply:
Dat is inderdaad voor de tests zo. Die zetten allemaal custom properties en je zou dat misschien niet willen in globale AppConstants instances. Maar als ik er een PropertyLoader van maak kan het wel.